### PR TITLE
refactor the event structure

### PR DESCRIPTION
**Pull request by mark86092**
Use jquery event to helping organize the structure

### CODE REVIEW

**Comment by barry800414** [comment]
I've done my review

File: src/js/main/index.js [hunk 1]
```diff
@@ -355,10 +356,31 @@ $(document).ready(function(){
         },
         minLength: 2,
         select: function(event, ui){
-            if(ga){
-                ga('send', 'event', 'LANDING_PAGE', 'company-query-autocomplete-select', ui['item']['company_id']);
-            }
-            $("#company_id").val(ui['item']['company_id']);
-        }
+            $company_query.trigger('autocomplete-select', ui.item.company_id);
+            $("#company_id").val(ui.item.company_id);
```
Comment (mark86092) [inline src/js/main/index.js]: seems not only typo, but also the id is changed

`#company_id` --> `#form-company-id`

Comment (mark86092) [reply src/js/main/index.js]: can you help update it ?

Comment (barry800414) [reply src/js/main/index.js]: np

**Pull request merged**